price_feeder/tasks.py

```diff
@@ -452,6 +471,9 @@ def task_price_feed(self, task=None, global_manager=None):
         # Accepted variation to write to blockchain
         is_in_range = price_variation_oracle >= decimal.Decimal(price_variation_write_blockchain)
 
+        # Accepted variation to re write to blockchain
+        re_post_is_in_range = price_variation_oracle >= decimal.Decimal(price_variation_re_write_blockchain)
```

Comment:
This comparison, needs to be with your last pushed price, not with the `price_variation_oracle` one.